Marlin/src/module/tool_change.cpp

```diff
@@ -221,6 +221,129 @@
 
 #endif // PARKING_EXTRUDER
 
+#if ENABLED(SWITCHING_TOOLHEAD)
+
+  inline void switching_toolhead_tool_change(const uint8_t tmp_extruder, const float fr_mm_s/*=0.0*/, bool no_move/*=false*/) {
+    if (!no_move) {
+
+      constexpr uint16_t angles[2] = SWITCHING_TOOLHEAD_SERVO_ANGLES;
+
+      const float toolheadposx[] = SWITCHING_TOOLHEAD_X_POS,
+                  placexpos = toolheadposx[active_extruder] + hotend_offset[X_AXIS][active_extruder],
+                   grabxpos = toolheadposx[tmp_extruder] + hotend_offset[X_AXIS][active_extruder];
```

Comment:
Since the `hotend_offset` pertains to the nozzle's position relative to another nozzle on the same carriage, I'm pretty sure that it should not be included here.